lib/galaxy/workflow/modules.py

```diff
@@ -227,17 +233,18 @@ class SubWorkflowModule( WorkflowModule ):
     # - Second pass actually turn RuntimeInputs into inputs if possible.
     type = "subworkflow"
     name = "Subworkflow"
-    default_name = "Subworkflow"
 
     @classmethod
     def from_dict( Class, trans, d, **kwds ):
         module = Class( trans )
         if "subworkflow" in d:
-            module.subworkflow = d["subworkflow"]
+            module.subworkflow = d[ "subworkflow" ]
         elif "content_id" in d:
-            content_id = d["content_id"]
-            module.subworkflow = SubWorkflowModule.subworkflow_from_content_id( trans, content_id )
-        module.label = d.get( "label", None ) or None
+            from galaxy.managers.workflows import WorkflowsManager
```

Comment:
Yup - this is my hack, it definitely indicates a problem with the namespace design, and it needs to be fixed someday. I did prefer to tuck it away in a well named helper function to keep the main logic of that function simpler.